src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayCost = 40;
+  const discountThreeDays = 20;
```

Comment:
For future: Don't hardcode some values that may change in the future in variable names, in your case it's numbers